tools/gen_node_cert.sh

```diff
@@ -317,9 +319,9 @@ main()
 
         #move origin conf to gm conf
         rm ${output_dir}/${conf_path}/node.nodeid
-        cp ${output_dir}/${conf_path} ${output_dir}/${gm_conf_path}/origin_cert -r
+        cp -r ${output_dir}/${conf_path} ${output_dir}/${gm_conf_path}/origin_cert
         #remove original cert files
-        rm ${output_dir:?}/${conf_path} -rf
+        rm -rf ${output_dir:?}/${conf_path}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5472013)